src/client/js/section-observer.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const classNameToObserve = 'section-transition'
+const classNameEntered = `${classNameToObserve}-entered`
+const cueIntervalDuration = 150
+const sectionThreshold = 0.1
```

Comment:
Percentage of the section that has to be in the viewport in order to trigger the appearance transition.